packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -337,7 +331,7 @@ const Dashboard = () => {
     return (
         <>
             <Helmet>
-                <title>{dashboardName || dashboard.name} - Lightdash</title>
+                <title>{dashboard.name} - Lightdash</title>
```

Comment:
is removing dashboardName not causing a side effect ? like the page is not updated if the title changes..

Reply:
I've renamed some dashboard and didn't see any issues

Reply:
I think initially there was a logic where there was a dashboard rename modal which just updated dashboard state but was not persisting (you would open a modal, rename dashboard, would see a temporal dash name instead of the original one and then save the dashboard). then the modal was replaced with update dashboard name/description and modal was taking part of the save process. so it was a useless peace of code